app/models/stripe_wrapper.rb

```diff
@@ -0,0 +1,46 @@
+module StripeWrapper
+
+  DEFAULT_CURRENCY = "usd"
+
+  class Charge
+    def initialize(status, responses = {})
+      @status = status
+      @customer = responses[:customer]
+      @charge = responses[:charge]
+      @error = responses[:error]
+    end
+
+    def self.create(options = {})
+      begin
+        customer = Stripe::Customer.create(
+          source: options[:token],
+          description: options[:customer]
+        )
```

Comment:
Hm yes, agreed. 

In the current state of the application, an account is always created when charging the signup fee, thus it will always be a new customer. But this makes the StripeWrapper::Charge class also include creation of the Stripe::Customer. I've refactored the StripeWrapper to have two separate classes for Charge and Customer.